test/python/unit/api/test_term.py

```diff
@@ -0,0 +1,789 @@
+###############################################################################
+# Top contributors (to current version):
+#   Makai Mann, Andres Noetzli
+#
+# This file is part of the cvc5 project.
+#
+# Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+# in the top-level source directory and their institutional affiliations.
+# All rights reserved.  See the file COPYING in the top-level source
+# directory for licensing information.
+# #############################################################################
+##
+
+
+
+import pytest
+import pycvc5
+from pycvc5 import kinds
+from pycvc5 import Sort, Term
+
+def assert_eq(x, y):
```

Comment:
Is this just here to match the C++ tests better? I'm not a huge fan of it tbh, because it doesn't add anything but now you can't pass a message like `assert x == y, "Something important"`. Furthermore, then people have to remember to use it (see line 35 where it's not used).